go.mod

```diff
@@ -4,5 +4,5 @@ go 1.14
 
 require (
 	github.com/hashicorp/terraform-plugin-docs v0.4.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.6.1
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.9.0
```

Comment:
This change was necessary to permit the provider to work with newer versions of Terraform when running tests.